lib/install/config/webpacker.yml

```diff
@@ -3,6 +3,7 @@
 default: &default
   source_path: app/javascript
   source_entry_path: /
+  manifest_path: app/javascript/manifest.json
```

Comment:
I think it's awkward to include a build-generated, not in git, file inside of `/app`